index.js

```diff
@@ -4,7 +4,7 @@ import Fab from './Fab';
 import Icon from './Icon';
 import IconButton from './IconButton';
 import IconToggle from './IconToggle';
-import Switch from './Switch';
+import Switch from './Switch/index';
```

Comment:
@cromefire why is this different than everything else?

Reply:
I don't know fixing it

Reply:
Must have been my IDE for some reason

Reply:
Fixed

Reply:
oh that's `index.js`, well fixing it

Reply:
Okay now it's fixed